lib/sdf-server/tests/service_tests/mod.rs

```diff
@@ -6,14 +6,14 @@ use axum::{
 use serde::{de::DeserializeOwned, Serialize};
 use tower::ServiceExt;
 
-mod change_set;
-mod component;
+// mod change_set;
+// mod component;
 mod crdt;
-mod functions;
-mod scenario;
-mod schema;
-mod secret;
-mod session;
+// mod functions;
+// mod scenario;
+// mod schema;
+// mod secret;
+// mod session;
```

Comment:
Temporary until the multiplexers are plumbed through the `sdf_test` macro. This PR will move out of drafts once that is done and working.